streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1470,19 +1485,24 @@ private void shouldDownGradeSubscriptionToVersion2(final Object upgradeFromValue
     public void shouldReturnInterleavedAssignmentWithUnrevokedPartitionsRemovedWhenNewConsumerJoins() {
         builder.addSource(null, "source1", null, null, null, "topic1");
 
-        final Set<TaskId> allTasks = Utils.mkSet(task0_0, task0_1, task0_2);
+        final Set<TaskId> allTasks = mkSet(task0_0, task0_1, task0_2);
+        final Map<TaskId, Integer> allTaskLags = mkMap(
+            mkEntry(task0_0, 0),
+            mkEntry(task0_1, 0),
+            mkEntry(task0_2, 0)
+        );
 
-        subscriptions.put(c1,
-            new ConsumerPartitionAssignor.Subscription(
+        subscriptions.put(CONSUMER_1,
+                          new ConsumerPartitionAssignor.Subscription(
                 Collections.singletonList("topic1"),
                 getInfo(UUID.randomUUID(), allTasks, Collections.emptySet(), null).encode(),
-                Arrays.asList(t1p0, t1p1, t1p2))
+                asList(t1p0, t1p1, t1p2))
```

Comment:
prop: indent args to Subscription? If the current approach is an established code style then feel free to disregard.

Same comment for next `subscriptions.put` below.

Reply:
Huh, I'm not sure how that happened. It's not an established code style.

Since it's in the middle of this giant test class anyway, though, I think I'll leave it alone, unless I get some CR comments that require a new commit.